react/features/welcome/components/WelcomePage.web.js

```diff
@@ -9,7 +9,11 @@ import { CalendarList } from '../../calendar-sync';
 import { RecentList } from '../../recent-list';
 import { SettingsButton, SETTINGS_TABS } from '../../settings';
 
-import { AbstractWelcomePage, _mapStateToProps } from './AbstractWelcomePage';
+import {
+    AbstractWelcomePage,
+    ROOM_NAME_VALIDATE_PATTERN_STR,
```

Comment:
style: all caps goes first

Reply:
Is this written somewhere where one can take a look?